k9mail/src/main/java/com/fsck/k9/provider/UnreadWidgetProvider.java

```diff
@@ -115,7 +124,11 @@ public static void updateWidget(Context context, AppWidgetManager appWidgetManag
             remoteViews.setTextViewText(R.id.unread_count, displayCount);
         }
 
-        remoteViews.setTextViewText(R.id.account_name, accountName);
+        if (folderName != null) {
+            remoteViews.setTextViewText(R.id.account_name, accountName + "/" + folderName);
```

Comment:
I've changed the id to title, but I'm not sure I understand the rest. Do you mean to allow the user to modify the title of the widget ?